fdbclient/BlobCipher.cpp

```diff
@@ -1092,7 +1092,7 @@ EncryptBlobCipherAes265Ctr::~EncryptBlobCipherAes265Ctr() {
 // DecryptBlobCipherAes256Ctr class methods
 
 DecryptBlobCipherAes256Ctr::DecryptBlobCipherAes256Ctr(Reference<BlobCipherKey> tCipherKey,
-                                                       Optional<Reference<BlobCipherKey>> hCipherKey,
+                                                       Optional<Reference<BlobCipherKey>> hCipherKeyOpt,
```

Comment:
maybe rebase with `main`?

Reply:
I think you're looking at an older commit